schematics/scully/src/add-post/index.ts

```diff
@@ -1,15 +1,14 @@
-import {
-  Rule, SchematicContext, SchematicsException, Tree
-  // apply, branchAndMerge, chain, mergeWith, move, Rule, SchematicContext, SchematicsException, Tree, url,
-} from '@angular-devkit/schematics';
+import { Rule, SchematicContext, SchematicsException, Tree } from '@angular-devkit/schematics';
 import {Schema} from './schema';
+import { strings } from '@angular-devkit/core';
 
 export default function(options: Schema): Rule {
   return (host: Tree, context: SchematicContext) => {
 
-    const name = options.title ? options.title : 'blog-X';
-    if (!host.exists(`./blog/${name}.md`)) {
-      host.create(`./blog/${name}.md`,
+    const name = options.name ? options.name : 'blog-X';
+    const namD = options.name ? strings.dasherize(options.name) : 'blog-X';
+    if (!host.exists(`./blog/${namD}.md`)) {
```

Comment:
@jorgeucano did you allow for people to put their blogs in a different directory besides 'blog`? If so, then we need to fix this code so that `blog` isn't hardcoded.

Reply:
The schematics for blog is only for the 'blog' folder & blog module.
If you need create a fileContent plugin in another folder (with different name) you need use the markdown schematics.
```
ng g @scullyio/init:markdown --name=test --slug=blah
```